torch/lib/ATen/Formatting.h

```diff
@@ -6,11 +6,10 @@
 
 namespace at {
 
-
-std::ostream& operator<<(std::ostream & out, IntList list);
-std::ostream& operator<<(std::ostream & out, Backend b);
-std::ostream& operator<<(std::ostream & out, ScalarType t);
-std::ostream& print(std::ostream& stream, const Tensor & tensor, int64_t linesize);
+ATen_CLASS std::ostream& operator<<(std::ostream & out, IntList list);
```

Comment:
Can we rename this ATen_EXPORT if we are going to use it to export functions?

Reply:
ATen_EXPORT is defined while building using cmake, so we have to use another name.

Reply:
For export functions, there's ATen_API. It can be used for export C-style functions.